src/components/ControlledFieldArray/ControlledFieldArray.js

```diff
@@ -39,19 +47,21 @@ const ControlledFieldArray = ({ name, label }) => {
               variant="filled"
               value={value || ""}
               onChange={onChange}
-              // error={!!error}
-              // helperText={error ? error.message : null}
+              error={error && !!error[index]}
+              helperText={
+                error && error[index] ? error[index]?.value?.message : null
```

Comment:
like this, helperText can only be used for errors and there would be no way to actually use it for helper text. 
Could do something like this so we have an option to show helper text too
```js

const helperText = getError(error) || field?.helperText
//.. 
<Controller helperText={helperText} />
```